@vates/generator-toolbox/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "@vates/generator-toolbox",
+  "version": "0.0.0",
+  "main": "dist/index.mjs",
+  "license": "MIT",
+  "private": false,
+  "type": "module",
+  "devDependencies": {
+    "@eslint/js": "^9.19.0",
+    "@tsconfig/node-lts": "^20.1.3",
+    "@tsconfig/recommended": "^1.0.7",
+    "@types/node": "^22.3.0",
+    "typescript": "^5.5.4",
```

Your `typescript` version seems to break CI.
this seems to fix the problem: `~5.6`
```suggestion
    "typescript": "~5.6",
```